src/samples/http/server-https.c

```diff
@@ -0,0 +1,139 @@
+/*
+ * This file is part of the Soletta Project
+ *
+ * Copyright (C) 2016 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file
+ * @brief HTTPS sample
+ *
+ * Basic HTTPS server. It uses a given certificate to crypto its contents.
+ *
+ * To test it:
+ *
+ * run ./server-https -p 8080 -d "Hello HTTPS" -c certificate_name.pem -k key_name.pem
+ *
+ * To see the usage help, -h or --help.
+ */
```

Comment:
Could we provide instructions/links to someone generate certificates, just for testing purposes?